akka-stream/src/main/scala/akka/stream/impl/io/TLSActor.scala

```diff
@@ -442,7 +442,10 @@ import akka.util.ByteString
     }
   }
 
-  override def receive = inputBunch.subreceive.orElse[Any, Unit](outputBunch.subreceive)
+  override def receive = inputBunch.subreceive.orElse[Any, Unit](outputBunch.subreceive).orElse {
+    case ActorGraphInterpreter.Snapshot =>
+      sender() ! StreamSnapshotImpl(self.path, Seq.empty, Seq.empty)
```

Comment:
The problem here is that the path does not say that this is a TLS stage which will probably lead to some further head scratching. Could we change the naming of TLSModule actors as well?